packages/map-template/src/components/Map/MapboxMap/MapboxMap.jsx

```diff
@@ -55,7 +57,7 @@ function MapboxMap({ onMapView, onPositionControl }) {
 
     // Add Floor Selector to the Map when ready.
     useEffect(() => {
-        if (mapsIndoorsInstance && mapView && !hasPositionControl) {
+        if (mapsIndoorsInstance && mapView && showPositionControl && !hasPositionControl) {
```

Comment:
Yes, I believe the kioskOriginLocationId prop on the component should react to changes, just like all our other props. It would be weird if exactly this one doesn't.

Reply:
Refactored the handling of showing and hiding the my position control button to a much simpler approach. Followed the same approach as the floor selector and venue selector, hiding them using CSS classes.

Reply:
👏